plugins/output-mongodb/src/main/scala/com/stratio/sparkta/plugin/output/mongodb/MongoDbOutput.scala

```diff
@@ -51,19 +51,20 @@ class MongoDbOutput(keyName: String,
 
   override val retrySleep = properties.getString("retrySleep", DefaultRetrySleep).toInt
 
-  override val idAsField = Try(properties.getString("idAsField").toBoolean).getOrElse(false)
```

Comment:
This function is removed from the mongo-spark-connector, I change the documentation.